Revised rule-based and unsupervised sections (#52)
greenelab/knowledge-graph-review: content/03.text-mining.md

Two databases DISEASES [@doi:10.1016/j.ymeth.2014.11.020] and STRING [@doi:10.1093/nar/gks1094] were populated using a co-occurrence scoring method on PubMed abstracts, which measured the frequency of co-mention pairs within individual sentences as well as the abstracts themselves.
This technique assumes that each individual co-occurring pair is independent from one another.
Under this assumption mention pairs that occur more than expected were presumed to implicate the presence of an association or interaction.

> This approach was able to identify 543,405 disease gene associations [@doi:10.1016/j.ymeth.2014.11.020] and 792,730 high confidence protein-protein interactions [@doi:10.1093/nar/gks1094] but is limited to only using PubMed abstracts.

Reviewer: ```suggestion
This approach identified 543,405 disease gene associations [@doi:10.1016/j.ymeth.2014.11.020] and 792,730 high confidence protein-protein interactions [@doi:10.1093/nar/gks1094] but is limited to only using PubMed abstracts.
```

Full text articles are able to dramatically enhance relationship detection [@doi:10.1371/journal.pcbi.1005962; @doi:10.1093/nar/gkt1207].
Westergaard et al. used a co-occurrence approach, similar to DISEASES [@doi:10.1016/j.ymeth.2014.11.020] and STRING [@doi:10.1093/nar/gks1094], to mine full articles for protein-protein interactions and other protein related information [@doi:10.1371/journal.pcbi.1005962].
The authors discovered that full text provided better prediction power than using abstracts alone, which suggests that future text mining approaches should consider using full text to increase detection power.

Unsupervised extractors often treat different biomedical relationships as multiple isolated problems.
An alternative to this perspective is to capture all different types at once.

> Clustering is an approach that accomplishes this concept of simultaneous extraction.

Reviewer: ```suggestion
Clustering is an approach that performs simultaneous extraction.
```
I think the meaning is unchanged here.
